labgrid/remote/client.py

```diff
@@ -820,13 +820,15 @@ def main():
     subparser = subparsers.add_parser('io',
                                       help="change (or get) a digital IO status")
     subparser.add_argument('action', choices=['high', 'low', 'get'])
+    subparser.add_argument('-v', '--verbose', action='count', default=0)
```

Comment:
Sure, I was not sure whether we prefer it for the individual subparsers here, since two already implement `--verbose` with different actions.